GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeViewModel.swift

```diff
@@ -21,7 +21,8 @@ extension SecureConversations {
         var environment: Environment
 
         var messageText: String = "" { didSet { reportChange() } }
-        var isAttachmentsAvailable: Bool = true { didSet { reportChange() } }
+        // By default attachments are not available, until site configurations are fetched.
```

Comment:
since this comment defines variable behavior, I think we can add one more slash and make this comment a doc for this var. wdyt?